### PR TITLE
Add new file

### PR DESCRIPTION
**Pull request by iwebtest**
Homework7

### CODE REVIEW

**Comment by iwebexpert** [comment]
Hi

**Comment by iwebtest** [comment]
Code

File: Igor/test.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Тестовый файл
+<script>
+console.log(1234);
```
Comment (iwebexpert) [inline Igor/test.html:3]: `console.log("12345");`

**Review by iwebexpert — CHANGES_REQUESTED** [review]
See comments

**Review by iwebexpert — APPROVED** [review]

**Pull request merged**